src/carousel.ts

```diff
@@ -1,10 +1,14 @@
+import Hammer from 'hammerjs';
```

Comment:
This is the correct place to import.  We generally don't want to depend on global variables for imports.